### ISSUE TITLE
Fix all activation-key and content-view tests where we are using redhat contents via manifest

### ISSUE THREAD

**Issue by sghai** [SatelliteQE/robottelo#1844]
Error Message
```
u'success' != u'warning'
- success
+ warning
 : [u'Import is the same as existing data\nOwner has already imported from another subscription management application.']
-------------------- >> begin captured logging << --------------------
root: DEBUG: NoSuchElementException: Could not locate element //div[contains(@class, 'jnotify-notification-error')].
--------------------- >> end captured logging << ---------------------
```
Stacktrace
```
  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/workspace/satellite6-automation@2/DISTRO/rhel66/ENDPOINT/ui/PROJ/upstream/PYTHON/System-CPython-2.7/tests/foreman/ui/test_contentviews.py", line 695, in test_cv_promote_rh_1
    self.setup_to_create_cv(session, cv_name, rh_repo=rh_repo)
  File "/home/jenkins/workspace/satellite6-automation@2/DISTRO/rhel66/ENDPOINT/ui/PROJ/upstream/PYTHON/System-CPython-2.7/tests/foreman/ui/test_contentviews.py", line 74, in setup_to_create_cv
    u'success', task['result'], task['humanized']['errors']
  File "/usr/lib64/python2.7/unittest/case.py", line 553, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib64/python2.7/unittest/case.py", line 964, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python2.7/unittest/case.py", line 450, in fail
    raise self.failureException(msg)
"u'success' != u'warning'\n- success\n+ warning\n : [u'Import is the same as existing data\\nOwner has already imported from another subscription management application.']\n-------------------- >> begin captured logging << --------------------\nroot: DEBUG: NoSuchElementException: Could not locate element //div[contains(@class, 'jnotify-notification-error')].\n--------------------- >> end captured logging << ----
```

**Comment by sghai** [comment]
Looks like here we are importing the manifest in same org again and agains so this issue appears.. I made some changes..will raise the PR soon.

**Comment by sghai** [comment]
Made changes to fix all affected tests...Testing the code changes..

**Comment by elyezer** [comment]
Cool, I'm looking forward it.

I'm adjusting tags for this issue.

**Comment by sghai** [comment]
PR merged #1872. Closing this now.. as jenkins is happy with all CV and ak tests :)
